hrstaffdir/tests/phpunit/WebTest/HRStaffDir/HRStaffDirProfile.php

```diff
@@ -24,7 +24,6 @@
  +--------------------------------------------------------------------+
 */
 
-require_once 'CiviTest/CiviSeleniumTestCase.php';
```

Comment:
Sorry I didn't make that clear on the wiki! Independent of the type of the test, if you run it with phpunit you will need both files.